src/game.cpp

```diff
@@ -7685,8 +7685,12 @@ void game::examine( const tripoint &examp )
                 return;
             }
         }
+        if( mon != nullptr && mon->has_flag( MF_MILKABLE ) ) {
+            add_msg( m_info, _( "This is a milkable %s." ), mon->name().c_str());
+            iexamine::milk_source( mon );
```

Comment:
Wouldn't that require to move all the pet menu and all that to monexamine.h/.cpp or monsterexamine.h/.cpp? I mean, I could try to do it, but I really don't know how much help I would need to make it work.